### PR TITLE
DOCSP-30553: Change streams

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-rust/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-30553>
Staging - <https://preview-mongodbnorareidy.gatsbyjs.io/rust/DOCSP-30553-change-streams/fundamentals/crud/read-operations/change-streams/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+Include Pre-images and Post-images
+----------------------------------
+
+You can configure the change event to contain or omit the following data:
+
+- The **pre-image**, a document that represents the version of the
+  document before the operation, if it exists
+- The **post-image**, a document that represents the version of the
+  document after the operation, if it exists
```
[the remark this answers is not in the document]
Comment (rachel-mack) [reply source/fundamentals/crud/read-operations/change-streams.txt:169]: Suggestion: You may also want to note that you can make inclusion of these images required (which can throw errors) or dependent on availability. You've demonstrated it in your examples, but I think stating it here in the intro section would be beneficial.

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: monitor, delta, code example 
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/crud/read-operations/change-streams.txt:17]: definitely meant to use 2 here, thanks for catching!

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+.. literalinclude:: /includes/fundamentals/code-snippets/crud/watch.rs
+   :language: rust
+   :dedent:
+   :start-after: start-create-coll
+   :end-before: end-create-coll
+
+You can change the pre-image and post-image option in an existing collection
+by running the ``collMod`` command from the MongoDB Shell. To learn how to
+perform this operation, see the entry on :manual:`collMod </reference/command/collMod/>`
+in the Server manual.
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/crud/read-operations/change-streams.txt:234]: Yes you can! I added that you can use the driver and linked to the Rust page on running commands

File: source/fundamentals/crud/read-operations/change-streams.txt [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+.. warning::
+
+   If you enabled pre-images or post-images on a collection, modifying
+   these settings with ``collMod`` can cause existing change streams on
+   that collection to terminate.
```
[the remark this answers is not in the document]
Comment (norareidy) [reply source/fundamentals/crud/read-operations/change-streams.txt:240]: Ran some code to confirm, but yeah it's only if you've enabled them in the past and then use collMod to turn them off

**Review by rachel-mack — APPROVED** [review]
LGTM

**Review by abr-egn — APPROVED** [review]
LGTM!

**Pull request merged**